frontend/src/components/page-lifeGroups/fiveEs.js

```diff
@@ -43,56 +43,69 @@ const fiveEInfo = [
   },
 ];
 
-const FiveEs = () => (
-  <div>
-    <h3 className="text-center text-[24px] font-semibold mb-10 leading-tighter">
-      HMCC has defined the five values of biblical community found in Acts
-      2:42-47 as follows:
-    </h3>
-    <div className="flex flex-col lg:flex-row gap-x-[44px] lg:mx-[2px] pb-[5.25rem] lg:pb-[6.25rem]  pl-8 pr-4 lg:px-0">
-      {fiveEInfo.map((item, i, index) => (
-        <span
-          key={`fiveEInfo-${index}`}
-          className="flex flex-col text-center items-center"
-        >
-          <Circle color={item.color} />
-          <div className="pt-5 pb-3 text-2xl font-bold lg:font-semibold leading-tighter">
-            {item.word}
-          </div>
-          <div className="flex flex-col">
-            <div
-              className={`text-base tracking-medium-wide font-medium ${textContainer}`}
-            >
-              {parse(item.text)}
+const FiveEs = ({ lgSignupsRef }) => {
+  return (
+    <div>
+      <h3 className="text-center text-[24px] font-semibold mb-10 leading-tighter">
+        HMCC has defined the five values of biblical community found in Acts
+        2:42-47 as follows:
+      </h3>
+      <div className="flex flex-col lg:flex-row gap-x-[44px] lg:mx-[2px] pl-8 pr-4 lg:px-0">
```

Comment:
i would like to have anything bigger than 8px we should use rem for people who zoom in or zoom out on the site.  Could update those that has greater than 8px to rem please?

Reply:
got it, some of these were here before so i just left them since they technically werent part of the issue. but in the future ill just change it when i see it.